.github/workflows/front.yml

```diff
@@ -24,6 +24,9 @@ jobs:
 
       - name: Install dependencies
         run: yarn --cwd front install
+      
+      - name: Check linter
+        run: yarn --cwd front eslint --ext .ts,.tsx src --max-warnings 320
```

Comment:
eslint already runs as part of the build process, see vite.config.ts
https://vite-plugin-checker.netlify.app/configuration/config.html 
it could be nice to not run checkers (which also includes tsc) if this is done separately here
Also, shouldn't also eslint run on .js and .jsx files?

Reply:
Fixed !